File: WowUp.Common/Enums/WowClientType.cs
using System.ComponentModel.DataAnnotations;

namespace WowUp.Common.Enums
{
    public enum WowClientType
    {
        [Display(Name = "Retail")]
        Retail,
        [Display(Name = "Classic")]
        Classic,
        [Display(Name = "Retail PTR")]
        RetailPtr,
        [Display(Name = "Classic PTR")]
        ClassicPtr,
        [Display(Name = "Beta")]
        Beta,
        None
    }
}

File: wowup-electron/src/common/warcraft/wow-client-type.ts
// Various client types that WoW has to offer
export enum WowClientType {
  Retail = 0,
  Classic,
  RetailPtr,
  ClassicPtr,
  Beta,
  ClassicBeta,
  ClassicEra,
  ClassicEraPtr,
  None,
}

// Grouping of the various clients into their expansions
export enum WowClientGroup {
  Retail,
  BurningCrusade,
  Classic,
  WOTLK,
}
